src/formatter/builder.js

```diff
@@ -25,14 +25,14 @@ export default class FormatterBuilder {
 
   static getConstructorByType(type, options) {
     switch (type) {
-      case 'event-protocol':
-        return EventProtocolFormatter
       case 'json':
         return JsonFormatter
       case 'progress':
         return ProgressFormatter
       case 'progress-bar':
         return ProgressBarFormatter
+      case 'protobuf':
```

Comment:
@mpkorstanje suggested we call the new formatter `message` formatter rather than `protobuf` formatter. I agree - it's more understandable for users who may not be familiar with protocol buffers (nor should they have to care - it's an implementation detail).

Reply:
Sounds good, done